nav2_msgs/msg/CostmapUpdate.msg

```diff
@@ -0,0 +1,11 @@
+# Update msg for Costmap containing the modified part of Costmap
+std_msgs/Header header
+
+uint32 x
+uint32 y
+
+uint32 size_x
```

Comment:
In my opinion, the names `width` and `height` are a bit confusing, since they do not clearly indicate the connection to the respective dimensions. Also, I took the naming convention from `CostmapMetaData.msg`